src/TConsole.cpp

```diff
@@ -550,7 +552,7 @@ TConsole::TConsole(Host* pH, ConsoleType type, QWidget* parent)
     layerCommandLine->setPalette(__pal);
 
     changeColors();
-    if (mType & (MainConsole|Buffer)) {
+    if (mType == MainConsole) {
```

Comment:
I am picking away at them as I find them. To me this is an *obvious* case where previous code did not consider the `Buffer` case and I think it will be safe to drop it from **here**... :boom: